content/en/tracing/trace_explorer/visualize.md

```diff
@@ -28,6 +28,8 @@ The default sort for spans in the list visualization is by timestamp, with the m
 
 The configuration of the columns is stored alongside other elements of your troubleshooting context in saved views.
 
+You may see spans without data under the "LATENCY BREAKDOWN" column (There are some examples pictured in the GIF above). This can happen if the trace is somehow malformed, for example if there is a loop in the trace or if the root span is missing. Sometimes, the root service may decide to drop the trace but a downstream service thinks it’s interesting to keep. This means that only the downstream-service spans are ingested, resulting in an incomplete trace. It would be misleading to display a latency breakdown for an incomplete trace, so in these cases, the data is omitted instead. 
```

Comment:
```suggestion
If the trace is somehow malformed it could result in spans without data under the "LATENCY BREAKDOWN" column. For example, if there is a loop in the trace or if the root span is missing, the root service decides to drop the trace but a downstream service decides to keep it. This means that only the downstream-service spans are ingested, resulting in an incomplete trace. It is misleading to display a latency breakdown for an incomplete trace, so the data is omitted. 
```